apps/hostmetrics.go

```diff
@@ -33,6 +36,10 @@ func (r MetricsReceiverHostmetrics) Type() string {
 }
 
 func (r MetricsReceiverHostmetrics) Pipelines(ctx context.Context) []otel.ReceiverPipeline {
+	return r.MergeWithProcessor(ctx, nil)
+}
+
+func (r MetricsReceiverHostmetrics) MergeWithProcessor(ctx context.Context, processor confgenerator.MetricsProcessor) []otel.ReceiverPipeline {
```

Comment:
This looks good! Followed this to update the PR. 
1. Changed `MergeMetricsProcessor` to `ModifyReceiver`, as discussed in the other comment; 
2. Changed `disableGPU` to be an exported property; our feature tracking uses recursive reflections on the fields - having a nonexported field here would cause the feature extraction to fail.

Reply:
2 is only relevant if we care about extracting this as a feature, though. Do we?

Reply:
Yes, this field does not need to be tracked; changed this back to `disableGPUMetrics` and modified feature tracking to skip unexported fields.